lms/static/sass/application-extend2.scss.mako

```diff
@@ -43,6 +43,7 @@
 @import 'elements/typography';
 @import 'elements/controls';
 @import 'elements/system-feedback';
+@import 'elements/dropdown';
```

Comment:
I'm not sure dropdown menus alone need their own Sass partial. This seems like its related to buttons and controls. I'd rather it be placed in either of the following: 

* added to the ``_controls.scss`` partial
* renamed to a new file called ``_buttons.scss`` or ``menus`` as it really is a type of either of those elements.

Your choice.